extension/content.js

```diff
@@ -3,7 +3,7 @@
 const path = location.pathname;
 const isDashboard = path === '/';
 const isRepo = /^\/[^/]+\/[^/]+/.test(path);
-const repoName = path.split('/')[2];
+let repoName = path.split('/')[2];
```

Comment:
Don't change this. It's a global and is meant to be the current repoName, not the remote.